From a07b3bdcdc34339733f7409978667742d162fa41 Mon Sep 17 00:00:00 2001
Message-ID: <a07b3bdcdc34339733f7409978667742d162fa41.1716242564.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1716242563.git.sharlatanus@gmail.com>
References: <cover.1716242563.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 19 May 2024 11:55:04 +0100
Subject: [PATCH 21/26] gnu: python-ewah-bool-utils: Update to 1.2.0.

* gnu/packages/python-compression.scm (python-ewah-bool-utils): Update to 1.2.0.

Change-Id: I7bc50249f2d7e97c64d23054ad176cf0e4af6258
---
 gnu/packages/python-compression.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/python-compression.scm b/gnu/packages/python-compression.scm
index 947dd189d3..96e607bfe9 100644
--- a/gnu/packages/python-compression.scm
+++ b/gnu/packages/python-compression.scm
@@ -200,13 +200,13 @@ (define-public python-cramjam
 (define-public python-ewah-bool-utils
   (package
     (name "python-ewah-bool-utils")
-    (version "1.1.0")
+    (version "1.2.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "ewah_bool_utils" version))
        (sha256
-        (base32 "11z9vv9m19370j762z100dqhy2ndjzkxk3w9z02lzwadbyibal3h"))))
+        (base32 "1bff3cv5m4n5pwqz0q90dy42vpyrrhylnrkcd49g1dlf3fs6r0pp"))))
     (build-system pyproject-build-system)
     (arguments
      (list
-- 
2.41.0

